pallets/parachain-staking/src/lib.rs

```diff
@@ -420,6 +420,28 @@ pub mod pallet {
 
 			weight
 		}
+
+		fn on_runtime_upgrade() -> Weight {
+			log::info!(target: "parachain-staking", "Initializing parachain-staking storage with config defaults");
```

Comment:
This migration is idempotent, so I don't think that I need to prevent it from being run multiple times though we should still remove it once it's no longer required.